File: bungeecord/src/main/java/gg/norisk/bungeecord/api/NrcChannelRegister.java
package gg.norisk.bungeecord.api;

import gg.norisk.core.common.Constants;
import net.md_5.bungee.api.plugin.Plugin;

public class NrcChannelRegister {
    public static void register(Plugin plugin) {
        plugin.getLogger().info("Registering '" + Constants.NRC_CHANNEL + "' channel...");
        var proxy = plugin.getProxy();
        proxy.registerChannel(Constants.NRC_CHANNEL);
        plugin.getLogger().info("Channel registration complete.");
    }
} 

File: bungeecord/src/main/kotlin/gg/norisk/bungeecord/api/NrcChannelRegister.kt
package gg.norisk.bungeecord.api

import gg.norisk.core.common.Constants.Companion.NRC_CHANNEL
import net.md_5.bungee.api.plugin.Plugin

object NrcChannelRegister {
    fun register(plugin: Plugin) {
        plugin.logger.info("Registering '$NRC_CHANNEL' channel...")
        val proxy = plugin.proxy
        proxy.registerChannel(NRC_CHANNEL)
        plugin.logger.info("Channel registration complete.")
    }
}
